documentation/agent/usage.md

```diff
@@ -35,27 +58,47 @@ by sending HTTP requests to the agent.
 
 Example:
 
-    $> fusioninventory-agent --server http://my.server/plugins/fusioninventory/ --daemon
+    $> fusioninventory-agent --server http://my.server --daemon
+
+That's the easiest mode to set up, offering (almost) complete control from a
+centralized point, fully compatible with all available agent features, and the
+most flexible in term of usage. However, that's also the most expensive in term
+of resources usage, and the less secure. And the more important: who controls
+the GLPI servers also controls all assets where an agent is installed, with
+ability to execute arbitrary code with full system privileges, which is after
+all the intended purpose.
 
 ## Half-managed mode
 
-This mode requires an external scheduling process to run the agent (cron, task
-scheduler, trainee, ...) periodically. There is no impact on memory and
-security between agent executions. The server still retains control over what
-the agent executes exactly, however, as the agent will ask it for every
-available module.
+This mode requires a local triggering mechanism to launch the agent. It may be
+a scheduling system (cron, task scheduler) to run it automatically and
+regularily, but it may as well be a user session start script, for instance. As
+a consequence, there is no perl interpreter permanently loaded in memory, only
+during agent execution, reducing memory impact. And there is no privileged
+process listening on a network port, which is better for security, but also
+prevents the server to trigger agent executions itself. But the server still
+retains control over execution plan, as the agent ask for its tasks list when
+run.
 
 Example:
 
-    $> fusioninventory-agent --server http://my.server/plugins/fusioninventory/
+    $> fusioninventory-agent --server http://my.server
```

Comment:
As said before, you should better keep the full URL in example to avoid any confusion.